app/helpers/event_helper.rb

```diff
@@ -1,2 +1,10 @@
 module EventHelper
+
+  def location_string location
```

Comment:
It's not possible to create an event that doesn't have a location using the admin interface as the select field for location doesn't have a blank option. That said, there is no validation on the model that will prevent one from being created via other means. I'd say that's the real issue as we are unlikely to ever announce an event without having a venue arranged.